core/arch/arm/kernel/rpc_io_i2c.c

```diff
@@ -0,0 +1,61 @@
+// SPDX-License-Identifier: BSD-2-Clause
+/*
+ * Copyright 2020 Foundries Ltd <jorge@foundries.io>
+ *
+ */
+#include <kernel/rpc_io_i2c.h>
+#include <kernel/thread.h>
+#include <mm/mobj.h>
+#include <optee_rpc_cmd.h>
+#include <string.h>
+
+#define I2C_BUFFER_LENGTH 512 /* enough for most use cases */
```

Comment:
Maybe make it configurable: `CFG_RPC_I2C_BUFFER_SIZE ?= 512` in **mk/config.mk**.

Reply:
good point. will do.